javascript/ql/src/semmle/javascript/RegularExpressions.qll

```diff
@@ -0,0 +1,30 @@
+/**
+ * Provides classes and predicates for working with regular expressions.
```

Comment:
Overly generic description.

Reply:
> For reference, the sibling Promises.qll contains the equally generic expression: "Provides classes for working with promises.

That seems appropriate, since that library is indeed meant to collect all our API for working with promises (as far as I remember, anyway).

The present library, however, doesn't seem to have a very clearly defined scope. As it stands, I would actually advocate merging it into `RegExp.qll`, making that library our one-stop shop for working with regexes.